Xamarin.PropertyEditing.Mac/PropertyEditorPanel.cs

```diff
@@ -97,8 +111,16 @@ public void Select (IEnumerable<object> selectedItems)
 			((ObservableCollectionEx<object>)SelectedItems).Reset (selectedItems);
 		}
 
+		public override void ViewDidChangeEffectiveAppearance ()
+		{
+			this.propertyArrangeMode.Font = HostResourceProvider.GetNamedFont (PropertyEditorControl.DefaultFontName, PropertyEditorControl.DefaultFontSize);
```

Comment:
Ran the project for the first time and `this.property*` all all null, so getting an NRE exception on start-up.